pkg/scheduler/framework/plugins/nodevolumelimits/csi.go

```diff
@@ -27,30 +27,30 @@ import (
 	"k8s.io/kubernetes/pkg/scheduler/nodeinfo"
 )
 
-// NodeVolumeLimits is a plugin that checks node volume limits
-type NodeVolumeLimits struct {
+// CSILimits is a plugin that checks node volume limits.
+type CSILimits struct {
 	predicate predicates.FitPredicate
 }
 
-var _ framework.FilterPlugin = &NodeVolumeLimits{}
+var _ framework.FilterPlugin = &CSILimits{}
 
-// Name is the name of the plugin used in the plugin registry and configurations.
-const Name = "NodeVolumeLimits"
+// CSIName is the name of the plugin used in the plugin registry and configurations.
+const CSIName = "CSILimits"
```

Comment:
I think the name should stay ```NodeVolumeLimits```. Once the vendor specific ones are deprecated, we will be left with a single generic one, and naming it ```NodeVolumeLimits``` from now is necessary because once this is GA it will be difficult to rename.

Reply:
ok, should struct stay NodeVolumeLimits, too?

Reply:
name changed

Reply:
just the name because it is the one that will practically be part of the API compatibility test, we can rename the struct later once it is the sole one.